frontend/src/components/pagedItemsTable/PagedItemsTable.tsx

```diff
@@ -251,7 +247,7 @@ function PagedItemsTable<T extends object>({
 						columns={columns}
 						rowKey={getRowKeyForDataObject}
 						scroll={{x: 300}}
-						onChange={sortByCallback}
+						onChange={useDebounce(sortByCallback)}
```

Comment:
Be careful about conditional rendering. `useDebounce` is a hook which must always be executed for each re-rendering.

Reply:
sure no problem! this is sound and clear

Reply:
no major changes either, all is working as it should.